pipelines/vars/katello_proxy.yml

```diff
@@ -0,0 +1,2 @@
+foreman_installer_additional_packages:
+  - foreman-installer-katello
```

Comment:
Don't we have a `foreman-proxy-content` meta package? Personally I'm ok with dropping that and fully depending on the installer though :)